### ISSUE TITLE
Breaking imports on "Failed to save bookmarks: title must not be empty"

### ISSUE THREAD

**Issue by howesteve** [go-shiori/shiori#153]
Hello,

I understand a bookmark must have a title. But does it need to stop importing (shiori import command)? I have thousands of bookmarks on hundreds of folders, while importing it doesn't even give a clue on what bookmark was the culprit. I'm cutting a lenghty log and adding only the last two lines:
```
Skip http://www.focalprice.com/: URL already exists
Failed to save bookmarks: title must not be empty
```

Thanks,
Howe

**Comment by RadhiFadlillah** [comment]
Hi @howesteve,

Nope, that shouldn't be happening. By default, if title doesn't exist Shiori will uses the URL as bookmark's title. I'll look at it as soon as possible.

**Comment by RadhiFadlillah** [comment]
It should be fixed by [018f332](https://github.com/go-shiori/shiori/commit/018f33228a57629abc7dade1c013b4f595717a61). Could you try it out again ?

**Comment by RadhiFadlillah** [comment]
Since it seems this issue has been solved, I will close it. Feel free to reopen it if needed.
